### DISCUSSION TITLE
[G7] SensorSanity: Sensor Raw Data Sanity Failure: -0.1279

### DISCUSSION THREAD

**Discussion by sennierer** [NightscoutFoundation/xDrip#2940] [Q&A]
Hi,
I am using a Roche Accu-Chek Guide to calibrate a Dexcom G7 sensor.
The Guide is linked via Bluetooth and the readings are imported correctly to xdrip, but when I click on use for calibration I get these error messages in the log:
```
SensorSanity: Sensor Raw Data Sanity Failure: -0.1279
Sensor data fails sanity test - Cannot Calibrate!
```
If I later on add the calibration manually it says that it is ignoring the calibration as an identical one has been already processed.
Is the calibration used even with these error logs?
Or do I need to switch to manually entering the calibrations.

I am using one of the latest nightly builds (cant find the exact commit hash in the app)

Thanks!

**Comment by Navid200** [comment]
G7 is not calibrated by xDrip.
xDrip submits the calibration to the device that does the calibration.
We have some legacy messages that re still showing and we need to clean up some of them.  But, you can ignore the messages.

**Comment by sennierer** [comment]
Ok, great. Thanks very much for the quick reply!
